scripts/windows/test/Run-E2ETest.ps1

```diff
@@ -265,6 +265,24 @@ Function GetImageArchitectureLabel
     Throw "Can't find image architecture label for $Architecture"
 }
 
+Function GetLongHaulDeploymentFilename
+{
+    if (GetImageArchitectureLabel -eq "amd64")
+    {
+        # Using versions without snitcher and influxdb, as they are currently not working in windows
+        return "long_haul_deployment.template.windows.json"
```

Comment:
Either you always return this value since we only run x64 now; or throw exception if architecture label is not amd64.

Reply:
Ah yes, added exceptions.